### PR TITLE
[4.0] [a11y] Remove redundant title from site components

### PR DESCRIPTION
**Pull request by brianteeman**
The title attribute is generally not very accessible. Keyboard users and touch screen users will never see it.

It is particularly useless to add a title which is the same as other text already present as that can result in a screen reader announcing the same text twice.

This PR removes the redundant title attribute where the same text is already displayed and removes/replaces one language string where it was not needed

NOTE this is just a first pass at removing the low hanging fruit and a further pr will be needed when the tooltips are reviewed

### CODE REVIEW

**Comment by wilsonge** [comment]
Overriding system tests as appear to have broken in drone at the moment

**Comment by brianteeman** [comment]
thanks

File: components/com_content/tmpl/category/blog_children.php [hunk 1]
```diff
@@ -28,7 +28,7 @@
 				<?php if ($lang->isRtl()) : ?>
 				<h3 class="page-header item-title">
 					<?php if ( $this->params->get('show_cat_num_articles', 1)) : ?>
-						<span class="badge badge-info tip hasTooltip" title="<?php echo HTMLHelper::_('tooltipText', 'COM_CONTENT_NUM_ITEMS_TIP'); ?>">
+						<span class="badge badge-info tip hasTooltip">
```
Comment (Quy) [inline components/com_content/tmpl/category/blog_children.php:31]: Remove ` tip hasTooltip`. 
Add `<?php echo Text::_('COM_CONTENT_NUM_ITEMS'); ?>&nbsp;`

File: components/com_content/tmpl/category/default_children.php [hunk 2]
```diff
@@ -28,7 +28,7 @@
 				<?php if ($lang->isRtl()) : ?>
 				<h3 class="page-header item-title">
 					<?php if ( $this->params->get('show_cat_num_articles', 1)) : ?>
-						<span class="badge badge-info tip hasTooltip" title="<?php echo HTMLHelper::_('tooltipText', 'COM_CONTENT_NUM_ITEMS_TIP'); ?>">
+						<span class="badge badge-info tip hasTooltip" title="<?php echo HTMLHelper::_('tooltipText', 'COM_CONTENT_NUM_ITEMS'); ?>">
```
Comment (Quy) [inline components/com_content/tmpl/category/default_children.php:31]: Remove `tip hasTooltip` and title attribute.
Add `<?php echo Text::_('COM_CONTENT_NUM_ITEMS'); ?>&nbsp;`

File: components/com_content/tmpl/category/default_children.php [hunk 3]
```diff
@@ -43,7 +43,7 @@
 				<h3 class="page-header item-title"><a href="<?php echo Route::_(ContentHelperRoute::getCategoryRoute($child->id, $child->language)); ?>">
 					<?php echo $this->escape($child->title); ?></a>
 					<?php if ( $this->params->get('show_cat_num_articles', 1)) : ?>
-						<span class="badge badge-info tip hasTooltip" title="<?php echo HTMLHelper::_('tooltipText', 'COM_CONTENT_NUM_ITEMS_TIP'); ?>">
+						<span class="badge badge-info tip hasTooltip" title="<?php echo HTMLHelper::_('tooltipText', 'COM_CONTENT_NUM_ITEMS'); ?>">
```
Comment (Quy) [inline components/com_content/tmpl/category/default_children.php:46]: Remove `tip hasTooltip` and title attribute.
Add `<?php echo Text::_('COM_CONTENT_NUM_ITEMS'); ?>&nbsp;`

**Pull request merged**